File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao24.py
numero = int(input("Digite um número inteiro: "))
numero_invertido = 0
numero_dividido = numero

while numero_dividido > 0:
    digito = numero_dividido % 10
    numero_invertido = numero_invertido * 10 + digito
    numero_dividido //= 10

print(f"O número {numero} invertido fica {numero_invertido}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao24.kt
fun main() {
    println("Digite um número inteiro: ")
    var numero = readLine()!!.toInt()
    var numeroInvertido = 0
    var alteraNumero: Int = numero
    while(alteraNumero > 0) {
        var digito = alteraNumero % 10
        numeroInvertido = numeroInvertido * 10 + digito
        alteraNumero = alteraNumero / 10
    }
    println("O numero ${numero} invertido é o numero ${numeroInvertido}")
}